packages/xrpl/src/models/methods/serverInfo.ts

```diff
@@ -158,6 +166,14 @@ export interface ServerInfoResponse extends BaseResponse {
        * cost.
        */
       load_factor_server?: number
+      /**
+       * The number of peer connections which were severed.
+       */
+      peer_disconnects?: string
```

Comment:
If these aren't on xrpl.org, there should be an issue opened over there to add these.

Reply:
https://github.com/XRPLF/xrpl-dev-portal/issues/1691